lang/en/admin.php

```diff
@@ -793,8 +793,10 @@
 $string['manageformats'] = 'Manage course formats';
 $string['manageformatsgotosettings'] = 'Default format can be changed in {$a}';
 $string['managelang'] = 'Manage';
-$string['questionbanks'] = 'Question bank plugins';
-$string['manageqbanks'] = 'Manage question bank plugins';
+$string['questionbanks'] = 'Question banks';
```

Comment:
the strings must be ordered alphabetically, this is not the correct position 'questionbanks'

Reply:
Moved in proper position